examples/write_video_uncompressed.py

```diff
@@ -0,0 +1,66 @@
+"""
+Demonstrates writing a video file to a DICOM dataset in an uncompressed format
+while keeping the data out of memory.
+"""
+import os
+import cv2
+from tempfile import TemporaryFile
+from pydicom import Dataset
+from pydicom.uid import UltrasoundMultiFrameImageStorage, generate_uid
+
+cine = cv2.VideoCapture("cine.mp4")
+mp4_file_size = os.stat("cine.mp4").st_size
+
+if not cine.isOpened():
+    raise RuntimeError("Could not open cine")
+
+file_meta = Dataset()
+file_meta.MediaStorageSOPClassUID = UltrasoundMultiFrameImageStorage
+file_meta.MediaStorageSOPInstanceUID = generate_uid()
+
+dataset = Dataset()
+dataset.file_meta = file_meta
+dataset.is_little_endian = True
+dataset.is_implicit_VR = True
```

Comment:
Shouldn't be needed with a transfer syntax set by the file meta

Reply:
Good to know!